docs/python_api/datasource/open_targets/_open_targets.md

```diff
@@ -7,3 +7,9 @@
     display: none;
   }
 </style>
+
+The Open Targets Platform is a comprehensive resource that aims to aggregate and harmonise various types of data to facilitate the identification, prioritisation, and validation of drug targets. Genomic data from Open Targets provides gene identification and genomic coordinates that are integrated into the gene index of our ETL pipeline.
```

Comment:
I'd include the text that goes from "Genomic data from Open Targets..." to "... for a reviewed protein product." as docstrings inside the `OpenTargetsTarget` class. This extract basically describes the target, not OT as a source.

Reply:
Thank you for this feedback! I've moved this text to the `OpenTargetsTarget` class docstring and I have updated the rest of the text I've got in this `_open_targets.md` file to focus more on OT as a source. I've still included a few sentences focusing on the genomic data, but happy to rewrite as needed. 

Also, I realised I'm not sure how much we want to write here so I'm also happy to expand if this is too short.